bundles/org.openhab.ui.basic/web-src/_theming.scss

```diff
@@ -7,6 +7,8 @@ body[data-theme="default"] {
 	--container-text-color: #616161;
 	--switch-on-track-bg: #9fa8da;
 	--border-color: #ccc;
+	--input-border-color: rgba(0,0,0,.12);
+	--input-undef-color: rgba(0,0,0,.26);
```

Comment:
Couldn't we use an existing variable for undef color ? Why a different color ? Should'nt it be simply the default color used for the Text widget for example ? I am not sure there is a specific color already defined for UNDEF value.

Reply:
See above. We could look for an existing color, but I do think it should be distinct from a real value.

Reply:
Not sure what to do with the undef color. The best match might be to also use the border color for this, but I am not sure.

Reply:
Let's keep it as you did.